### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by supreme5678**

### CODE REVIEW

File: web/src/components/ProjectCard.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import { CheckCircleOutlined, SyncOutlined } from '@ant-design/icons'
+import { Avatar, Card, Col, Row, Tag, Typography } from 'antd'
+import React from 'react'
+const { Title, Text } = Typography
```
Comment (tonklaartisan) [inline web/src/components/ProjectCard.tsx:4]: Move into the function

File: web/src/components/WelcomeCard.tsx [hunk 2]
```diff
@@ -0,0 +1,124 @@
+import { CloseCircleOutlined } from '@ant-design/icons'
+import { Card, Typography } from 'antd'
+import React, { useState } from 'react'
+
+const { Text } = Typography
```
Comment (tonklaartisan) [inline web/src/components/WelcomeCard.tsx:5]: Move into the function

File: web/src/components/WelcomeCard.tsx [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+function WelcomeCard({ name, task, project }: any) {
+  const [disable, setDisable] = useState(false)
+
+  function onCloseClick() {
+    setDisable(true)
+  }
+  return disable ? null : (
```
Comment (tonklaartisan) [inline web/src/components/WelcomeCard.tsx:13]: Must return a DOM, not `null`

File: web/src/components/WelcomeCard.tsx [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+        style={{ fontSize: '24px', color: '#fff' }}
+      />
+      <div className="ml-24 relative z-50">
+        <div>
+          <Text className="text-white text-2xl font-bold z-30">
+            {!name ? `Hi!` : `Hi ${name} !`}
+          </Text>
+        </div>
+        <div>
+          <Text className="text-white text-lg font-light">{`You have  `}</Text>
```
Comment (tonklaartisan) [inline web/src/components/WelcomeCard.tsx:30]: Why not use `<span className="mr-2">` instead of white space?

File: web/src/pages/Dashboard.tsx [hunk 5]
```diff
@@ -1,42 +1,117 @@
-import { Card, Col, Row } from 'antd'
+import { FundProjectionScreenOutlined, ProfileOutlined, TeamOutlined } from '@ant-design/icons'
+import { Card, Col, Row, Typography } from 'antd'
 import React from 'react'
-import { LayoutDashboard } from '../components/DashboardComponent'
+import { LayoutDashboard, ProjectCard, WelcomeCard } from '../components/DashboardComponent'
 import '../styles/main.css'
 
+const { Title, Text } = Typography
```
Comment (tonklaartisan) [inline web/src/pages/Dashboard.tsx:7]: Move into the function

File: web/src/pages/Dashboard.tsx [hunk 6]
```diff
@@ -1,42 +1,117 @@
...
+
+const data = [
```
Comment (tonklaartisan) [inline web/src/pages/Dashboard.tsx:9]: Move into the function

File: web/src/components/ProjectCard.tsx (hunk 1 above)
Comment (supreme5678) [reply web/src/components/ProjectCard.tsx:4]:
```suggestion
const { Title, Text } = Typography
```

This line right ?

Comment (tonklaartisan) [reply web/src/components/ProjectCard.tsx:4]: Every global variables outside the component.

Comment (supreme5678) [reply web/src/components/ProjectCard.tsx:4]: Ok krub

**Pull request merged**